docs/VMR-Feature-Band-Support-Work-Plan.md

```diff
@@ -0,0 +1,92 @@
+# Feature Band Support: Work Plan
+
+This document builds on top of [Managing SDK Bands](https://github.com/dotnet/dotnet/blob/main/docs/VMR-Managing-SDK-Bands.md) with a concrete work plan for providing feature band support in the VMR. Please read that doc as a prerequisite.
+
+Document convention:
+* There can be a variable number of feature band releases (e.g. 10.0.1xx, 10.0.2xx, 10.0.3xx, etc). For purposes of this document, we mainly care about the distinction between the 1xx branch and all the other non-1xx feature band branches. For simplicity, 2xx will be used as a notation to mean any non-1xx feature band branch.
+
+## Plan Details
+
+### Branch Management
+
+The 2xx branch sources should accurately reflect what gets produced in the output. Since the shared component are only being produced in the 1xx branch, those associated repos will be excluded from the 2xx branch upon creation.
+
+There should be tooling to help with this exclusion process for the VMR maintainers. Similarly, the associated projects in the `repo-projects` directory should be deleted along with updating the appropriate `RepositoryReference` usages.
```

Comment:
It might be better to not delete the repo-projects but condition them based on the feature band. This would allow to forward merge changes from 10.0.2xx to 11.0.1xx (the dotnet/sdk model basically) without a ton of conflicts.